uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/ConfigurationsEditorEndpointTest.java

```diff
@@ -125,84 +129,53 @@
 	 */
 	@Parameters
 	public static Collection<String> setupData() {
-
-		new WorkbenchShell();
-		ProjectFactory.newProject(PROJECT_NAME).deploymentType(STANDALONE).runtimeType(KARAF)
-				.version(CAMEL_2_17_0_REDHAT_630187).template(CBR_SPRING).create();
-		for (IViewReference viewReference : WorkbenchPartLookup.getInstance().findAllViewReferences()) {
-			if (viewReference.getPartName().equals("Welcome")) {
-				final IViewReference iViewReference = viewReference;
-				Display.syncExec(new Runnable() {
-					@Override
-					public void run() {
-						iViewReference.getPage().hideView(iViewReference);
-					}
-				});
-				break;
-			}
-		}
-
+		createProject();
 		new CamelProject(PROJECT_NAME).openCamelContext(CONTEXT);
-		List<String> endpoints = CamelEndpointDialog.getEndpoints();
-		return endpoints;
+		CamelEditor.switchTab("Configurations");
+		return ConfigurationsEditor.getEndpoints();
 	}
 
-	/**
-	 * Utilizes passing parameters using the constructor
-	 * 
-	 * @param template
-	 *            a Global element - Endpoint
-	 */
-	public ConfigurationsEditorEndpointsTest(String element) {
-		this.element = element;
+	public ConfigurationsEditorEndpointTest(String endpoint) {
+		this.endpoint = endpoint;
+		this.endpointName = label(endpoint);
+		path = new String[] { ROOT, this.endpointName, " (Endpoint)" };
 	}
 
-	/**
-	 * Prepares test environment
-	 */
 	@BeforeClass
 	public static void setupResetCamelContext() {
+		new WorkbenchShell().maximize();
+		new CleanWorkspaceRequirement().fulfill();
```

Comment:
Is it necessary? You use class annotation CleanWorkspaceRequirement, therefore it should be fulfilled before tests ;-).